src/Adapters/CommonShared/SdkVersionUtils.cs

```diff
@@ -0,0 +1,27 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="SdkVersionUtils.cs" company="Microsoft">
+// Copyright (c) Microsoft Corporation. 
+// All rights reserved.  2013
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Microsoft.ApplicationInsights.Implementation
+{
+    using System;
+    using System.Linq;
+    using System.Reflection;
+
+    internal class SdkVersionUtils
+    {
+        internal static string GetSdkVersion(string versionPrefix)
+        {
+            string versionStr = typeof(SdkVersionUtils).Assembly.GetCustomAttributes(false)
+                    .OfType<AssemblyFileVersionAttribute>()
+                    .First()
+                    .Version;
+
+            Version version = new Version(versionStr);
+            return (versionPrefix ?? string.Empty) + version.ToString(3) + "-" + version.Revision;
```

Comment:
Other than a minorly different implementation of this last line what is the difference between this method and GetExpectedSdkVersion earlier in the review?